src/util/fields.py

```diff
@@ -1159,6 +1159,17 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
         "models": tuple(only_models) if only_models else (),
     }
 
+    # update order in ir.model
+    if column_exists(cr, "ir_model", "order") and only_models:
+        cr.execute(
+            rf"""
```

Comment:
1. Do not format queries ever with the standard Python formatting tools. Use `turmat_query`
2. f-strings are invalid for Python2, all utils are Python2 compatible.
```suggestion
            r"""
```